_posts/daiblej/2020-09-22-daiblej-treasure-hunt-turtle.md

```diff
@@ -9,7 +9,7 @@ title: "Janet's treasure hunt turtle"
 
 :turtle:
 This is an interesting exercise. I think the most difficult part is the animations function. However, it's already there. It is a good reference for me to define my own function.
-The first thing I need to do is avoiding the wrong input. I think input x and y are very similar so I tried to define a function to repeat the input section. However, I have a problem with this phrase. I don't know how to repeat the try and except function until the user input a correct number between -100 to 100. Actually, after I finished the exercise, I checked others' ideas. I found someone put a "continue" in the while loop which allows me continuously inputting the number until it is between -100 and 100. However, when I tried to define this in my "tina_input(coordinate)" function, it said, "'continue' outside the loop". I haven't figured out the right answer yet.
-Another thing I learned is the abs(). With abs, the distance calculation becomes much easier. I also give notification to their distance input. It makes the whole code more user-friendly to our users. I'm also surprised about the user_has_won. With the hint true and false, I quickly know how to end the loop.
+The first thing I need to do is avoiding the wrong input. I think input x and y are very similar so I tried to define a function to repeat the input section. However, I have a problem with this phrase. I don't know how to repeat the `try` and `except` function until the user input a correct number between -100 to 100. Actually, after I finished the exercise, I checked others' ideas. I found someone put a `continue` in the while loop which allows me continuously inputting the number until it is between -100 and 100. However, when I tried to define this in my `tina_input(coordinate)` function, it said, "`continue` outside the loop". I haven't figured out the right answer yet.
+Another thing I learned is the `abs()`. With abs, the distance calculation becomes much easier. I also give notification to their distance input. It makes the whole code more user-friendly to our users. I'm also surprised about the `user_has_won`. With the hint true and false, I quickly know how to end the loop.
```

Comment:
I think there need to be three backslashes! so it would be ```abs()``` for all the times you want to show code :)

Reply:
``````abs()``````